src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,36 @@
 package core.basesyntax;
 
 public class MainApp {
```

Comment:
You should create method "public static void main(String[] args)" in this class and run your logic in this method

Reply:
I create method "public static void main(String[] args)" and realise my logic in this method.
But I have a BUILD FAILURE: File does not end with a newline. [NewlineAtEndOfFile], please see in chat

Reply:
I create method "public static void main(String[] args)" and run my logic in this method